<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Экспоненциальное распределение</string>
    </property>
    <widget class="QDoubleSpinBox" name="spinbox_intensivity_oa">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>121</width>
       <height>26</height>
      </rect>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>115</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Интенсивность :</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Генератор</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>ОА</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>261</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Равномерное распределение</string>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>115</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Интенсивность :</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>34</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>СКО:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinbox_intensivity_gen">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>121</width>
       <height>26</height>
      </rect>
     </property>
     <property name="maximum">
      <double>99999.990000000005239</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinbox_intensivity_gen_range">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>121</width>
       <height>26</height>
      </rect>
     </property>
     <property name="maximum">
      <double>99999.990000000005239</double>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>168</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Время моделирования:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="spinbox_time_model">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>99999999.989999994635582</double>
    </property>
    <property name="value">
     <double>100.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_model">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>559</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Моделировать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_graph">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>559</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Построить график</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>471</width>
      <height>151</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>Загрузка системы</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Среднее время завки в очереди</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Количество обработанных заявок</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Общее время моделирования</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Теоретич.</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Фактич.</string>
     </property>
    </column>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>561</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>561</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
